alignak/external_command.py

```diff
@@ -1680,8 +1670,9 @@ def del_all_host_comments(self, host):
         :type host: alignak.objects.host.Host
         :return: None
         """
-        for comment in host.comments:
-            self.del_host_comment(comment)
+        comments = host.comments.keys()
```

Comment:
I do not understand... host.comments is a dictionnary, right? Indexed with the comment uuid, right? So, why not keeping the previous code that should suit for this?